mappings/net/minecraft/world/gen/decorator/LeaveVineTreeDecorator.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_4661 net/minecraft/world/gen/decorator/LeaveVineTreeDecorator
+	METHOD method_23467 addHangingVine (Lnet/minecraft/class_3747;Lnet/minecraft/class_2338;Lnet/minecraft/class_2746;Ljava/util/Set;Lnet/minecraft/class_3341;)V
```

Comment:
```suggestion
	METHOD method_23467 addVine(Lnet/minecraft/class_3747;Lnet/minecraft/class_2338;Lnet/minecraft/class_2746;Ljava/util/Set;Lnet/minecraft/class_3341;)V
```

Reply:
But it is a hanging vine and not a vine. What if in the future there is just a vine and it gets added ?

Reply:
If it happens in the future, we can deal with it then. No point in worrying about that now.